packages/x-date-pickers-pro/src/StaticNextDateRangePicker/StaticNextDateRangePicker.tsx

```diff
@@ -36,7 +42,7 @@ const StaticNextDateRangePicker = React.forwardRef(function StaticNextDateRangeP
     showToolbar: defaultizedProps.showToolbar ?? displayStaticWrapperAs === 'mobile',
   };
 
-  const { renderPicker } = useStaticPicker({
+  const { renderPicker } = useStaticPicker<DateRange<TDate>, TDate, DateOrTimeView, typeof props>({
```

Comment:
I would pass `'day'` instead of `DateOrTimeView` here like for the other date range pickers
And remove the added views to the lookup

`DateOrTimeView` is wrong since we will never have time view here
And `DateView` would feel misleading since we don't really have month / year view